From 46c2eff5adca7dc309f077ec65faf95b95c47b43 Mon Sep 17 00:00:00 2001
From: "Miss Islington (bot)"
 <31488909+miss-islington@users.noreply.github.com>
Date: Thu, 11 Jul 2019 07:17:08 -0700
Subject: [PATCH 02/19] bpo-34369: make kqueue.control() docs better reflect
 that timeout is positional-only (GH-9499)

(cherry picked from commit 79042ac4348ccc09344014f20dd49401579f8795)

Co-authored-by: Tal Einat <taleinat@gmail.com>
---
 Doc/library/select.rst | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Doc/library/select.rst b/Doc/library/select.rst
index 7a080457a0..b68b32486a 100644
--- a/Doc/library/select.rst
+++ b/Doc/library/select.rst
@@ -291,13 +291,14 @@ Kqueue Objects
    Create a kqueue object from a given file descriptor.
 
 
-.. method:: kqueue.control(changelist, max_events[, timeout=None]) -> eventlist
+.. method:: kqueue.control(changelist, max_events[, timeout]) -> eventlist
 
    Low level interface to kevent
 
-   - changelist must be an iterable of kevent object or ``None``
+   - changelist must be an iterable of kevent objects or ``None``
    - max_events must be 0 or a positive integer
-   - timeout in seconds (floats possible)
+   - timeout in seconds (floats possible); the default is ``None``,
+     to wait forever
 
 
 .. _kevent-objects:
-- 
2.20.1 (Apple Git-117)

